### ISSUE TITLE
Export `StreamApiException` to outside the package

### ISSUE THREAD

**Issue by pedrohsampaioo** [GetStream/stream-feed-flutter#133]
**Is your feature request related to a problem? Please describe.**

I need to handle with some exceptions from the GetStream, example:
```dart
StreamApiException (StreamApiException{body: {detail: Reaction does not exist, status_code: 404, code: 16, exception: DoesNotExistException, duration: 1.31ms, more_info: https://getstream.io/docs/api_error_responses}, jsonData: null, status: 404, code: null})
```
but the package doesn't export the exception `StreamApiException`.

**Describe the solution you'd like**

Export https://github.com/GetStream/stream-feed-flutter/blob/master/packages/stream_feed/lib/src/core/exceptions.dart on https://github.com/GetStream/stream-feed-flutter/blob/master/packages/stream_feed/lib/src/core/index.dart .

**Describe alternatives you've considered**
Nop...

**Comment by sachaarbonel** [comment]
Hey @pedrohsampaioo, thanks for opening an issue this will be available in 0.3.0 #134

**Comment by sachaarbonel** [comment]
This feature was released
